src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx

```diff
@@ -789,7 +789,7 @@ const formatTime = (hours: number, minutes: number) => {
 };
 
 const formatDate = (dateValue: string) => {
```

Comment:
Nit: I question the naming of this function – it's doing precisely the opposite of "formatting" a date, isn't it? Taking a date that's in a particular expected format, and parsing it into an object…

```suggestion
const parseDate = (dateValue: string) => {
```